ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -55,11 +56,12 @@ Verify Pod Toleration
 
 Verify RHODS User Groups
     [Documentation]    Verify User Configuration after the upgrade
-    [Tags]      Upgrade
-    ${admin}        Set Variable        ${payload[0]['spec']['groupsConfig']['adminGroups']}
-    ${user}     Set Variable        ${payload[0]['spec']['groupsConfig']['allowedGroups']}
-    Should Be Equal As Strings      '${admin}'      'rhods-admins,rhods-users'
-    Should Be Equal As Strings      '${user}'       'system:authenticated'
+    [Tags]      Upgrade     Operator
+    Get Auth Cr Config Data
+    ${admin}                Set Variable            ${AUTH_PAYLOAD[0]['spec']['adminGroups']}
+    ${user}                 Set Variable            ${AUTH_PAYLOAD[0]['spec']['allowedGroups']}
```

Comment:
I think instead of reading the expected values from the cluster itself, we should impost the expected truth for the test.
So, if in the pre-upgrade tests we set 2 specific groups as admins, we should check in post upgrade that those groups are still set (in either UI and CRD or both)

cc: @jstourac @manosnoam

Reply:
and those are different from the ones set during pre-upgrade https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot#L78 

The test should check the current user group set in the Auth CR matches the ones we set during pre-upgrade no?


>I think instead of reading the expected values from the cluster itself,


yeah this was wrong sorry

Reply:
yes, it should check basically what is set on Auth CR